### PR TITLE
Adding weekly tpr and vaccination coverage to final data

### PR DESCRIPTION
**Pull request by henryls1**
Created a pull request to add TPR and Vaccination coverage data to the ELR spreadsheet. 

Process is as follows:

1. Data to be appended are saved in separate .rds to `final_data.rds` in `assemble_data.Rmd`
2. Data are loaded in `elr_review.Rmd`, and merged with final data, before saving in the final spreadsheet

### CODE REVIEW

File: report_sources/elr_review.Rmd [hunk 1]
```diff
@@ -586,12 +597,17 @@ out_final <- out_full %>%
...
     epi_classification = classif,
     growth_rate_lower = r_lower,
-    growth_rate_upper = r_upper,
-    growth_rate_tpr = r_tpr)
+    growth_rate_upper = r_upper
+    # growth_rate_tpr = r_tpr
+  )
+
+out_final <- out_final %>% 
+     left_join(elr_extras_final, by = "iso3")
```
Comment (thibautjombart) [inline report_sources/elr_review.Rmd:610]: It looks like this is the first time 'elr_extras_final' is defined, which causes the following error on my end when compiling the Rmd:
```
label: unnamed-chunk-18
Quitting from lines 572-612 (elr_review.Rmd) 
Error in is.data.frame(y) : object 'elr_extras_final' not found
```

**Review by thibautjombart — CHANGES_REQUESTED** [review]
Thanks a lot for this! Looks like one object is missing or might have been renamed, hopefully a small fix?

**Comment by henryls1** [comment]
My bad on that erorr - all fixed now @thibautjombart

**Review by thibautjombart — APPROVED** [review]
Works for me. Document compiles fine and the outputs make sense, though I think the tpr reported in the leaderboard of the report may still be the 'old' one. @nsbatra there are now quite a few new columns in the exported xlsx, can you check if you want to keep all of them or just a subset?

**Pull request merged**